src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -13395,6 +13395,9 @@ public enum Architecture
         LoongArch64 = 6,
         Armv6 = 7,
         Ppc64le = 8,
+        Riscv32 = 9,
+        Riscv64 = 10,
+        Riscv128 = 11,
```

Comment:
```suggestion
        RiscV64 = 9,
```

per the approved API https://github.com/dotnet/runtime/issues/73437#issuecomment-1716122572

Reply:
Thank you!